src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewItemAutomationPeer.cs

```diff
@@ -0,0 +1,317 @@
+﻿// MUX Reference TreeViewItemAutomationPeer.cpp, commit 46f9da3
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Uno.UI.Helpers.WinUI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using TreeView = Microsoft.UI.Xaml.Controls.TreeView;
+using TreeViewItem = Microsoft.UI.Xaml.Controls.TreeViewItem;
+using TreeViewNode = Microsoft.UI.Xaml.Controls.TreeViewNode;
+using TreeViewSelectionMode = Microsoft.UI.Xaml.Controls.TreeViewSelectionMode;
+
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	/// <summary>
+	/// Exposes TreeViewItem types to Microsoft UI Automation.
+	/// </summary>
+	public partial class TreeViewItemAutomationPeer : ListViewItemAutomationPeer, IExpandCollapseProvider
+	{
+		/// <summary>
+		/// Initializes a new instance of the TreeViewItemAutomationPeer class.
+		/// </summary>
+		/// <param name="owner">The TreeViewItem control instance to create the peer for.</param>
+		public TreeViewItemAutomationPeer(TreeViewItem owner) : base(owner)
+		{
+		}
+
+		// IExpandCollapseProvider
+
+		/// <summary>
+		/// Gets a value indicating the expanded or collapsed state of the associated TreeViewItem.
+		/// </summary>
+		public ExpandCollapseState ExpandCollapseState
+		{
+			get
```

Comment:
Codacy found an issue: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)